<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>switcher_window</class>
 <widget class="QDialog" name="switcher_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:
rgb(255, 255, 255);</string>
  </property>
  <widget class="QPushButton" name="Off">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border: 2px solid rgb(51,51,51);
	border-radius: 5px;	
	color:rgb(255,255,255);
	font-family:&quot;Calibri&quot;;
	font-size: 15px;
	background-color: rgb(51,51,51);
}
QPushButton:hover {
	border: 2px solid rgb(0,143,150);
	background-color: rgb(0,143,150);
}
QPushButton:pressed {	
	border: 2px solid rgb(0,143,150);
	background-color: rgb(51,51,51);
}

QPushButton:disabled {	
	border-radius: 5px;	
	border: 2px solid rgb(112,112,112);
	background-color: rgb(112,112,112);
}</string>
   </property>
   <property name="text">
    <string>Off</string>
   </property>
  </widget>
  <widget class="QPushButton" name="On">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border: 2px solid rgb(51,51,51);
	border-radius: 5px;	
	color:rgb(255,255,255);
	font-family:&quot;Calibri&quot;;
	font-size: 15px;
	background-color: rgb(51,51,51);
}
QPushButton:hover {
	border: 2px solid rgb(0,143,150);
	background-color: rgb(0,143,150);
}
QPushButton:pressed {	
	border: 2px solid rgb(0,143,150);
	background-color: rgb(51,51,51);
}

QPushButton:disabled {	
	border-radius: 5px;	
	border: 2px solid rgb(112,112,112);
	background-color: rgb(112,112,112);
}</string>
   </property>
   <property name="text">
    <string>On</string>
   </property>
  </widget>
  <widget class="QComboBox" name="rooms_list">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>50</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
	border: 2px solid rgb(51,51,51);
	border-radius: 5px;	
	color:rgb(255,255,255);
	font-family:&quot;Calibri&quot;;
	font-size: 15px;
	background-color: rgb(51,51,51);
}

QComboBox:hover {
	border: 2px solid rgb(0,143,170);
	border-radius: 5px;	
	color:rgb(255,255,255);
	background-color: rgb(0,143,170);
}

QComboBox:!editable, QComboBox::drop-down:editable {
	background: rgb(51,51,51);
}

QComboBox:!editable:on, QComboBox::drop-down:editable:on {
    background:rgb(51,51,51);
}

QComboBox:on { /* shift the text when the popup opens */
    padding-top: 3px;
    padding-left: 4px;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 15px;

    border-left-width: 1px;
    border-left-color: darkgray;
    border-left-style: solid; /* just a single line */
    border-top-right-radius: 5px; /* same radius as the QComboBox */
    border-bottom-right-radius: 5px;
}

QComboBox::down-arrow {
    image: url(icons/1x/arrow.png);
}

QComboBox::down-arrow:on { /* shift the arrow when popup is open */
    top: 1px;
    left: 1px;
}

QComboBox::drop-down {
    background:rgb(51,51,51);
}

</string>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
  </widget>
  <widget class="QSlider" name="horizontalSlider">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider::groove:horizontal {
    height:5px;
    background: rgb(51,51,51);
}

QSlider::handle:horizontal {
    background:rgb(0,143,170);
    width: 10px;
margin:-8px 0
}

QSlider::add-page:horizondal {
    background:rgb(51,51,51);
}

QSlider::sub-page:horizondal {
    background:rgb(51,51,51);
}</string>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>3</number>
   </property>
   <property name="sliderPosition">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
   <property name="invertedControls">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>100</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value" stdset="0">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>-60</x>
     <y>0</y>
     <width>451</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="reset">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border: 2px solid rgb(51,51,51);
	border-radius: 5px;	
	color:rgb(255,255,255);
	font-family:&quot;Calibri&quot;;
	font-size: 15px;
	background-color: rgb(51,51,51);
}
QPushButton:hover {
	border: 2px solid rgb(0,143,150);
	background-color: rgb(0,143,150);
}
QPushButton:pressed {	
	border: 2px solid rgb(0,143,150);
	background-color: rgb(51,51,51);
}

QPushButton:disabled {	
	border-radius: 5px;	
	border: 2px solid rgb(112,112,112);
	background-color: rgb(112,112,112);
}</string>
   </property>
   <property name="text">
    <string>RESET</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
